controllers/articles.controller.js

```diff
@@ -10,3 +11,13 @@ exports.getArticleById = (req, res, next) => {
       next(err);
     });
 };
+
+exports.getArticles = (req, res, next) => {
+  fetchArticles()
+    .then((allArticles) => {
+      res.status(200).send({ allArticles });
```

Comment:
`allArticles` is fine, I'd encourage you go with an even simpler key of `articles`